gatling/user-files/utils/documents.json

```diff
@@ -0,0 +1,3166 @@
+{
```

Comment:
What's the point of having a file with the same document repeated over and over?
Either handle duplication through code, or use an actually meaningful set of data.

Reply:
I need a json array with hundreds json documents to test routs like mCreate. no matter if documents are the same

Reply:
You still don't need to store all these occurrences in a file.
Just build an array:

```
const foo = {foo: 'bar'};
const bar = [];

for (let i = 0; i < 10000; i++) { 
  bar.push(foo); 
}
```

Reply:
right, fixed